diff -rupN linux-source-3.2.0.orig/arch/arm/mach-omap2/board-overo.c linux-source-3.2.0/arch/arm/mach-omap2/board-overo.c
--- linux-source-3.2.0.orig/arch/arm/mach-omap2/board-overo.c	2012-12-09 15:38:47.998911265 -0500
+++ linux-source-3.2.0/arch/arm/mach-omap2/board-overo.c	2012-12-09 15:42:35.130907002 -0500
@@ -500,6 +500,12 @@ static struct gpio overo_bt_gpios[] __in
 	{ OVERO_GPIO_BT_NRESET, GPIOF_OUT_INIT_HIGH,	"lcd bl enable" },
 };
 
+static void scout_power_off(void)
+{
+	if (gpio_request_one(88, GPIOF_OUT_INIT_HIGH, "Power off") < 0)
+		printk(KERN_ERR "Unable to get kill power GPIO\n");
+}
+
 static void __init overo_init(void)
 {
 	int ret;
@@ -554,6 +560,8 @@ static void __init overo_init(void)
 	else
 		printk(KERN_ERR "could not obtain gpio for "
 					"OVERO_GPIO_USBH_CPEN\n");
+
+	pm_power_off = scout_power_off;
 }
 
 MACHINE_START(OVERO, "Gumstix Overo")
